core/src/View/MDSpan/Kokkos_MDSpan_Accessor.hpp

```diff
@@ -307,6 +307,13 @@ class ReferenceCountedDataHandle {
   pointer m_handle = nullptr;
 };
 
+template <class T>
+struct IsReferenceCountedDataHandle : std::false_type {};
+
+template <class ElementType, class MemorySpace>
+struct IsReferenceCountedDataHandle<
+    ReferenceCountedDataHandle<ElementType, MemorySpace>> : std::true_type {};
+
```

Comment:
Consider also introducing the corresponding helper variable template (not formatted)
```suggestion

template <class T>
constexpr bool IsReferenceCountedDataHandleV = IsReferenceCountedDataHandle<T>::value;

```

Reply:
Done